core/trino-main/src/main/java/io/trino/cost/ComparisonStatsCalculator.java

```diff
@@ -167,9 +166,8 @@ public static PlanNodeStatsEstimate estimateExpressionToExpressionComparison(
             case GREATER_THAN_OR_EQUAL:
             case IS_DISTINCT_FROM:
                 return PlanNodeStatsEstimate.unknown();
-            default:
```

Comment:
can you add a not to codestyle README that we sholud avoid using `default` clause in `switch` statements?

Reply:
Added (you can preview this with https://github.com/trinodb/trino/compare/56a4c7c18c8163e663a8c5bc5a08b4f5a6b2e4e3..af78d32adc26574782cd6bb6aeb7f35495ad06de)